src/app/interfaces.ts

```diff
@@ -162,6 +162,52 @@ export interface IUserTagCategory {
     tags: IUserTag[];
 }
 
+export interface IComplaintTag {
+    id: string;
+    tag_uri: string;
+}
+
+export interface IComplaintComment {
+    id: string;
+    time: string;
+    text: string;
+    commented_by: IUserProfile;
+    reported_date: string;
+}
+
+export interface IComplaint {
+    id: string;
+    created_by: IUserProfile;
+    description: string;
+    suggestions: string;
+    location_details: string;
+    report_date: string;
+    reported_date: string;
+    status: string;
+    status_color: string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'status_color' is not in camel case.](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)